testhelpers/org.eclipse.n4js.tests.helper/src/org/eclipse/n4js/N4JSStandaloneTestsModule.java

```diff
@@ -28,6 +28,12 @@
  */
 @XpectGuiceModule
 public class N4JSStandaloneTestsModule extends BaseTestModule {
+
+	/** Constructor enables JS support */
+	public N4JSStandaloneTestsModule() {
```

Comment:
this is causing JS mode being enabled in lots of places. Also transitively via `N4JSInjectorProvider` and others.
In essence we are testing product with JS support and deliver product with not JS  support.
Could we narrow the scope enabling JS mode?

Reply:
If needed, have it in separate packages or even bundle?

Reply:
> In essence we are testing product with JS support and deliver product with not JS support.

-> Yes, that's exactly what we want.